arch/arm64/configs/eve_defconfig

```diff
@@ -1230,5 +1230,9 @@ CONFIG_SOFTLOCKUP_DETECTOR=y
 CONFIG_BOOTPARAM_SOFTLOCKUP_PANIC=y
 CONFIG_WQ_WATCHDOG=y
 CONFIG_SCHEDSTATS=y
+CONFIG_FUNCTION_PROFILER=y
+CONFIG_STACK_TRACER=y
+CONFIG_FTRACE_SYSCALLS=y
+CONFIG_BLK_DEV_IO_TRACE=y
```

Comment:
@christoph-zededa , did you run `make savedefconfig`  after editing your changes? For me it also adds the CONFIG_RELAY:

```diff
--- build/defconfig	2024-07-18 09:30:13.270323120 +0000
+++ arch/arm64/configs/eve_defconfig	2024-07-18 09:25:15.632827928 +0000
@@ -41,6 +41,7 @@ CONFIG_CGROUP_BPF=y
 CONFIG_NAMESPACES=y
 CONFIG_USER_NS=y
 CONFIG_CHECKPOINT_RESTORE=y
+CONFIG_RELAY=y
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_EXPERT=y
 CONFIG_SGETMASK_SYSCALL=y
```

Reply:
I added this change as a separate commit as it is also in `eve-kernel-arm64-v6.1.38`.